liblangutil/ErrorReporter.cpp

```diff
@@ -81,6 +82,16 @@ bool ErrorReporter::hasExcessiveErrors() const
 	return m_errorCount > c_maxErrorsAllowed;
 }
 
+bool ErrorReporter::hasError(unsigned long long _errorId) const
```

Comment:
Isn't this just `ErrorId`?

```suggestion
bool ErrorReporter::hasError(ErrorId _errorId) const
```

Reply:
`ErrorId` is a struct and I thought that `unsigned long long` would be more convenient since we could just write the error code number (instead of `ErrorId{number}`. 
What would we prefer?

Reply:
Definitely the struct. More type-safe and you can probably still use `{number}` and get the struct automatically constructed in places where spelling out its name is redundant.

Reply:
Right. Fixed it.